website_and_docs/content/blog/2022/end-of-year-review.md

```diff
@@ -0,0 +1,50 @@
+description: >
+  Looking back at our achievements from 2022
+---
+
+It was an amazing year here at the Selenium project, and we wanted to remind you of some of the great things we accomplished.
+
+In terms of releases, we continued to build on top of Selenium 4 which was launched in October 2021. This year we
+published [4.2.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.2.0),
+[4.3.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.3.0), [4.4.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.4.0),
+[4.5.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.5.0), [4.6.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.6.0)
+and [4.7.0](https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.7.0).
+These releases included improvements, bug fixes, removal of deprecated functionality, improved error handling,
+updates of the underlying libraries, support for the latest DevTools/BiDi
+versions to keep you in sync with new browser releases, but also new features.
+
+We introduced the first version (which we call beta) of the embedded Selenium Manager. The purpose of this feature is to
+help you manage your driver binaries without having to manually update them yourself each time a new version is released.
+We also introduced observability in the Docker-Selenium images. And we completed the implementation of the virtual
+authenticator across all languages. But these are only some of the highlights of what we released. To get the big picture for each
+language we support, you can check out their corresponding Changelog pages: [Java](https://github.com/SeleniumHQ/selenium/blob/trunk/java/CHANGELOG),
+[Python](https://github.com/SeleniumHQ/selenium/blob/trunk/py/CHANGES), [Ruby](https://github.com/SeleniumHQ/selenium/blob/trunk/rb/CHANGES),
+[Javascript](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/CHANGES.md),
+[DotNet](https://github.com/SeleniumHQ/selenium/blob/trunk/dotnet/CHANGELOG).
+
+In terms of events and gatherings, in July our Selenium India Conference took place, this time online. It kicked off with
+an 8 talk pre-conference event, namely Selenium Lite. We then had a full day of workshops, followed by 2 days of conference.
+We had over 50 speakers and over 800 participants from around the world. The talks focused on Selenium, the Selenium ecosystem,
+automation, methodologies and best practices. In case you missed some of these talks, you can still watch them
+[here](https://www.youtube.com/playlist?list=PL9Z-JgiTsOYRfoG_mcRBlTUIFPIknhQ6S).
+
+We also held 2 test automation summits: one in San Francisco and one in Berlin. During these workshops the participants,
+together with some of the maintainers of frameworks from the Selenium ecosystem, built their Selenium based projects,
+added tests, new features and fixed bugs. During the first summit the participants focused on the BiDi
```

Comment:
Link to spec - https://w3c.github.io/webdriver-bidi/

Reply:
Link to the blog article - https://www.selenium.dev/blog/2022/test-automation-summit/
Also, mention that Google, Microsoft, Mozilla and Apple all participated.